src/Lecture2.hs

```diff
@@ -101,7 +131,8 @@ spaces.
 
 🕯 HINT: look into Data.Char and Prelude modules for functions you may use.
 -}
-dropSpaces = error "TODO"
+dropSpaces :: String -> String
+dropSpaces = filter (not.isSpace).take 1024
```

Comment:
`take 1024` looks like a magic constant 😄 
I know it passes the tests but the function won't work on huge lists.

Have a look at functions `dropWhile` and `takeWhile` to implement a lazier solution without string trimming 🦥